pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml

```diff
@@ -215,6 +215,13 @@ spec:
                     call:
                       description: Name of the function to apply the kprobe spec to.
                       type: string
+                    ignoreOn:
```

Comment:
nit, I'd go with just 'ignore' but feel free to keep

Reply:
makes sense! changed it to `ignore:`